sdk/src/sysvar/rent.rs

```diff
@@ -49,6 +49,16 @@ pub fn create_account(lamports: u64, rent_calculator: &RentCalculator) -> Accoun
     .unwrap()
 }
 
+use crate::account::KeyedAccount;
```

Comment:
@jackcmay - is this going to cause heartburn for the bpf sdk?

Reply:
Not more then it already does in the rest of the sysvars ;-)